### ISSUE TITLE
Post: New Tricks I Learned from the Azure Data Studio Code Editor Tutorial

### ISSUE THREAD

**Issue by LitKnd** [LitKnd/littlekendracomments#33]
Comments left here will automatically appear on the post at https://www.littlekendra.com/2021/10/07/trying-out-azure-data-studio-quickstart/

**Comment by francesco1119** [comment]
And you can also connect to PostgreSQL from Azure Data Studio

**Comment by LitKnd** [comment]
@francesco1119  yes! I will say that I have found the extension for Postgres connections to be a bit tricky to use in the past. I used it for some demos and it was the kind of thing where I had to do things in an exact specific order to get it to work consistently -- otherwise I'd hit some issues where my connection wouldn't work and I'd have to restart Azure Data Studio. It's possible that these issues have been resolved since then, but it does look like the extension is still in "preview".
